lib/chutney/TorNet.py

```diff
@@ -1300,7 +1334,8 @@ def getNodeDirInfoStatusPattern(self, dir_format):
            the ed25519 key. (Or RSA block matching, which is hard.)
```

Comment:
More cleanup:
- The microdesc format is now implemented, so we can delete that comment.
- We need to explain what returning "None" means for this function. I can make that change. I should have done it when I wrote the function.